internal/core/ports/room_ports.go

```diff
@@ -24,11 +24,23 @@ package ports
 
 import (
 	"context"
+	"sync"
 	"time"
 
+	"github.com/topfreegames/maestro/internal/core/entities"
 	"github.com/topfreegames/maestro/internal/core/entities/game_room"
 )
 
+type RoomManager interface {
```

Comment:
@caiordjesus what do you think about separating the port types with comments as we did on operations ports?
```
// Primary ports (input, driving ports)
...
// Secondary ports (output, driven ports)
...
```